client/post-editor/post-editor.jsx

```diff
@@ -417,8 +417,8 @@ export const PostEditor = React.createClass( {
 				page.redirect( utils.getEditURL( post, site ) );
 			}
 			this.setState( postEditState, function() {
-				if ( didLoad || this.state.isLoadingAutosave ) {
-					this.refs.editor.setEditorContent( this.state.post.content, { initial: true } );
+				if ( this.editor && ( didLoad || this.state.isLoadingAutosave ) ) {
+					this.editor.setEditorContent( this.state.post.content, { initial: true } );
 				}
 
 				if ( this.state.isLoadingAutosave ) {
```

Comment:
yeah I wasn't sure if an error was warranted here or not. it didn't seem at glance that it was actually a problem for this to occur. it simply wouldn't update the editor, but since the editor isn't mounted any longer no harm done

thoughts?